sdk/servicebus/service-bus/README.md

```diff
@@ -39,39 +39,25 @@ You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your t
 
 ### Authenticate the client
 
-Interaction with Service Bus starts with an instance of the [ServiceBusClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient) class. You can instantiate
-this class using one of the 3 static methods on it
+Interaction with Service Bus starts with an instance of the [ServiceBusClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient) class.
 
-- [createFromConnectionString](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#createfromconnectionstring-string--servicebusclientoptions-)
+You can instantiate this class using its constructors:
+
+- [Create using a connection string](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#servicebusclient-string--servicebusclientoptions-)
   - This method takes the connection string to your Service Bus instance. You can get the connection string
     from the Azure portal.
-- [createFromTokenProvider](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#createfromtokenprovider-string--tokenprovider--servicebusclientoptions-)
-  - This method takes the host name of your Service Bus instance and your custom implementation of the [TokenProvider](https://github.com/Azure/amqp-common-js/blob/master/lib/auth/token.ts) interface.The host name is of the format `name-of-service-bus-instance.servicebus.windows.net`.
-- [createFromAADTokenCredentials](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#createfromaadtokencredentials-string--applicationtokencredentials---usertokencredentials---devicetokencredentials---msitokencredentials--servicebusclientoptions-)
+- [Create using a TokenCredential](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#servicebusclient-string--tokencredential---servicebusclientoptions-)
   - This method takes the host name of your Service Bus instance and a credentials object that you need
-    to generate using the [@azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth)
+    to generate using the [@azure/identity](https://www.npmjs.com/package/@azure/identity)
     library. The host name is of the format `name-of-service-bus-instance.servicebus.windows.net`.
```

Comment:
Host name doesn't include `sb://` ?

Reply:
I dont think so..

Reply:
Not in these cases. It does when you use a connection string but this is just the FQDN of the service bus instance.